docs/configuration/commons.md

```diff
@@ -249,8 +249,14 @@ Multiple sets of rates can be added to each frontend, but the time periods must
 ```
 
 In the above example, frontend1 is configured to limit requests by the client's ip address.  
-An average of 100 requests every 10 seconds is allowed and an average of 5 requests every 3 seconds.  
-These can "burst" up to 200 and 10 in each period respectively. 
+A sustained rate of 100 requests every 10 seconds (10 req/s) is allowed for rateset1, and 5 requests every 3 seconds (~1.67 req/s) for rateset2.  
+In addition, these can "burst" up to 200 and 10 in each period respectively.
+
+Another way to see the above parameters, is along the [leaky bucket](https://en.wikipedia.org/wiki/Leaky_bucket) analogy:
+for rateset1, the size of the bucket is 200 drops, and it is leaking at a rate of 10 drop/s.
+If the incoming rate of drops falling into the bucket gets high enough that the bucket gets filled,
+any subsequent drop overflows out of the bucket (i.e. discarded request).
+That situation holds until the incoming rate gets low enough again for the water level in the bucket to go down.
```

Comment:
```suggestion
This situation holds until the incoming rate remains low enough for the water level in the bucket to go down.
```

Reply:
hmm I don't think "remains" works here. we're in the state where the incoming rate is too high (requests are getting discarded), and this sentence describes how we're getting back to a state where the incoming rate is lower than what we allow in.

Reply:
The idea being that just not sending requests for a millisencond will not set the rate to 0, and allow all requests.

The incoming rate has to remain low enough for long enough that the bucket stops overflowing.

Reply:
how about the new wording I edited your suggestion into?

Reply:
Looks good to me!

Reply:
@dtomcej cool. suggestion committed.